state/pruner.go

```diff
@@ -178,6 +178,24 @@ func (p *Pruner) SetABCIResRetainHeight(height int64) error {
 	return err
 }
 
+// GetApplicationRetainHeight is a convenience method for accessing the
+// GetApplicationRetainHeight method of the underlying state store.
+func (p *Pruner) GetApplicationRetainHeight() (int64, error) {
```

Comment:
These getters are added such that #1154 can access these methods without needing to access the underlying state store.